Measurements/Workbook_Measurements.ipynb

```diff
@@ -470,9 +470,9 @@
     "Like in the previous solution, we are looking for the index Q where the two bit strings differ. \n",
     "Let's define a function `FindFirstSuperpositionDiff()` which searches for an index Q which has 2 properties:\n",
     "\n",
-    "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`, and the same for all arrays in `bits2`.  \n",
+    "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`, and is the same for all arrays in `bits2`.  \n",
```

Comment:
This reads to me like those values should be the same in both arrays, which will get confusing once we get to item 2... How about "and the same holds for all arrays in bits1"?

Reply:
Took me a while to make sense of this sentence and from what I understood (as a layman with no real background in programming or mathematics) the corresponding values in both bit strings need to be the same to satisfy this test.
I suppose my suggestion can be ignored, if it seems alright as it is.

Reply:
2nd item says that they have to be 0 in all elements of the first array and 1 in all elements of the second one or vice versa, but not the same across both arrays. How about splitting the 1st item into two, one for each array, and spelling it out completely? Something like "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`. If this is not the case, you cannot be sure that measuring the corresponding qubit will always return the same result. 2. The value of all arrays in `bits2` at the index Q is either `true` or `false`."

(I agree if it takes the task author some time to figure out the correct interpretation, it can be clarified :-))